protocol/messenger_communities.go

```diff
@@ -2305,21 +2305,44 @@ func (m *Messenger) ShareCommunity(request *requests.ShareCommunity) (*Messenger
 	if err := request.Validate(); err != nil {
 		return nil, err
 	}
+
+	var messageStatusLinkPreview common.StatusLinkPreview
 	response := &MessengerResponse{}
 
 	community, err := m.communitiesManager.GetByID(request.CommunityID)
 	if err != nil {
 		return nil, err
 	}
 
+	communityURL, err := m.ShareCommunityURLWithChatKey(community.ID())
```

Comment:
This should be `ShareCommunityURLWithData`.
Even though we don't use the shared URL data in desktop/mobile, we should still use those in case they will be opened in status-web someday.